### ISSUE TITLE
[Feature Request] Add failed scores to Recent Tab with a checkmark option or similar

### ISSUE THREAD

**Issue by GrilledCheeese** [ppy/osu-web#3651]
Just like osuplus does:
![old website osuplus](https://i.gyazo.com/300f5e48172367b588ede3a4456e55ab.png)
Current website without it:
![new website does not have it](https://i.gyazo.com/b9c45a0dddd16f85e56c509b1ab67aa9.png)

**Comment by peppy** [comment]
Sure, don't see harm in this. I'm guessing it should default to hidden?

**Comment by GrilledCheeese** [comment]
Yes default to hidden, to save possible load time for those that aren't going to use it anyway
